src/main/java/com/kevin/tiertagger/TierTagger.java

```diff
@@ -46,18 +49,20 @@ public static Text appendTier(PlayerEntity player, Text text) {
         return text;
     }
 
+//  Ooh_Netiyiy
     @Nullable
-    private static MutableText getPlayerTier(String username) {
+    public static MutableText getPlayerTier(String username) {
         if (tiers.containsKey(username)) {
             String foundTier = tiers.get(username);
             MutableText tier = Text.of(foundTier).copy();
-            if (username.equals("Ooh_Netiyiy")) {
-                tier.styled(s -> s.withColor(TextColor.parse("#A020F0")));
-            } else {
-                int color = getTierColor(foundTier);
-                tier.styled(s -> s.withColor(color));
+
+            if(username.equals("Ooh_Netiyiy")){
+                tier.styled(s -> s.withColor((int) ((tick % 60))));
+                return tier.append(": ");
             }
-            return tier;
+            int color = getTierColor(foundTier);
+            tier.styled(s -> s.withColor(color));
+            return tier.append(": ");
```

Comment:
you should append a reset here, to keep the formatting of the original chat message